doc/manual/index.adoc

```diff
@@ -1,14 +1,17 @@
-= Introduction to stillsuit
+= stillsuit user manual
 Tim Gilbert <tim@workframe.com>
-v0.1, 2018-01-26
+v0.10, 2018-05-04
 :sectanchors:
+:toc: left
+:source-highlight: true
 
 `stillsuit` is a library intended to be used with lacinia and datomic.
```

Comment:
I think you need to do more here to explain the purpose of stillsuit. Not from a high-level "How" perspective, but more from a high-level "What and Why" perspective.

Reply:
Added a little more detail here